tests/DefaultsTest.py

```diff
@@ -98,6 +101,18 @@ def test_004_workerclass(self):
         self.assertTrue(task.default("distant_worker") is WorkerSsh)
         task_terminate()
 
+        dname = make_temp_dir()
```

Comment:
temp dir should be explictly deleted.

Temp files are automatically deleted when the object is garbage collected though.

Reply:
Having the temp file automatically go away but the directory not was confusing me.  I switched to creating a regular file so I can safely blow away the entire temp directory.